NSS-Day-Cohort-49/giffygram-skateboard-bulldogs
# login

#### Changes Made
1. Modified file `login.js` to include all functionality to login and see the home page.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout lrc-start
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > See that when logging in using existing credentials (USERNAME: bob@smith.com PASSWORD: bob) then the user should see "GiffyGram" .
    > Example: When user does clicks Login, then the user should see GiffyGram  should happen.
4. View code file.
    > Confirm login.js modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
